conf/entrypoint.sh

```diff
@@ -32,6 +33,14 @@ setup_prod_db() {
     python manage.py migrate
 }
 
+start_app() {
+    cd /code
+    python manage.py collectstatic --noinput
+    chown www-data:www-data -R /var/www/static/
+    /usr/bin/uwsgi --ini /code/conf/uwsgi.ini
```

Comment:
We should use supervisor or dumb-init here instead of just running uwsgi direclty.